backend/src/main/java/gov/cdc/usds/simplereport/idp/repository/LiveOktaRepository.java

```diff
@@ -185,12 +163,19 @@ public Optional<OrganizationRoleClaims> createUser(
             .filter(g -> groupNamesToAdd.contains(g.getProfile().getName()))
             .map(Group::getId)
             .collect(Collectors.toSet());
-
-    UserBuilder.instance()
-        .setProfileProperties(userProfileMap)
-        .setGroups(groupIdsToAdd)
-        .setActive(active)
-        .buildAndCreate(_client);
+    try {
+      UserBuilder.instance()
+          .setProfileProperties(userProfileMap)
+          .setGroups(groupIdsToAdd)
+          .setActive(active)
+          .buildAndCreate(_client);
+    } catch (ResourceException e) {
+      if (e.getMessage()
```

Comment:
in a couple other places I'm throwing the `ConflictingUserException` if the message contains the Okta error message, hopefully that addresses: https://github.com/CDCgov/prime-simplereport/pull/5165#discussion_r1093805771

Reply:
This looks great! But I think it'd also be helpful to have an `else` block here (in case the `ResourceException` thrown isn't related to the duplicate user issue.) In that case, maybe just throw as an `IllegalStateException` or `IllegalGraphQlArgumentException`?

Reply:
Done!